src/containers/Login/index.test.js

```diff
@@ -0,0 +1,32 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import Login from '.';
+
+describe('<Login />', () => {
+  beforeEach(() => {
+    fetch.resetMocks();
+  });
+
+  it('renders the <Login /> component', () => {
+    const wrapper = shallow(<Login history={{ push() {} }} />);
+    expect(wrapper.contains(<h1>Authors' Haven</h1>));
+  });
+  it('Test handleSocialLogin is succesfull', () => {
+    fetch.mockResponseOnce(JSON.stringify({ user: {} }));
+    const wrapper = shallow(<Login history={{ push() {} }} />);
+    wrapper.instance().handleSocialLogin(
+      {
+        _provider: 'google',
+        _token: {
+          accessToken: 'ya29.GlsQBifRfrJ6lGYcM_0apRg8Tw8GWhtMZgY3dhZo8Msfa…8un9jFfxn0_4s6MBt9uss55R1AYJw',
```

Comment:
This looks like a real token, was there no other way to do it using a random token